File: 2022/python/daily-template/src/part2.py
from dotenv import load_dotenv
import os

load_dotenv()

filename = os.getenv('TEXTS_PATH')+'{{day}}.txt'
with open(filename, 'r') as file:
    data = file.read().strip()


def part2(data: str) -> int:
    result = 0

    return result


if __name__ == "__main__":
    print(part2(data))

File: 2023/rust/daily-template/src/bin/part2.rs
use {{crate_name}}::part2::process;
use miette::Context;

fn main() -> miette::Result<()> {
    let file = include_str!("../../../../day-01.txt");
    let result = process(file).context("process part 2")?;
    println!("{}", result);
    Ok(())
}